README.md

```diff
@@ -1,2 +1,32 @@
-# HelloMicroverse1
-Hello Microverse Task
+
+# Project Name: Hello Microverse
+
+> Hello Microverse Task 
+
+## Authors
+
+👤 **Richard W.K Chileya**
+
```

Comment:
- Good work so far, but there is an issue with your readme, Please make it more professional, presently it is missing things like the languages used, please make use of this [template](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) and add what is needed for this project.